src/utils/userManagement.js

```diff
@@ -0,0 +1,52 @@
+// Get user details from auth0 API. 
+'user strict';
+
+const config = require('../config');
+const logger = require('../logger').getLogger(module);
+const ManagementClient = require('auth0').ManagementClient;
+const request = require('co-request');
+
+function* getUserDetails(uuid) {
+  const token = yield getApiToken();
+  const auth0 = new ManagementClient({
+    token: token,
+    domain: config.get('AUTH0_DOMAIN')
+  });
+
+  return auth0
+    .getUsers({
+      fields: 'name,email,phone', // Use detauks field names to get from API.
+      q: 'app_metadata.dorbel_user_id: ' + uuid // Query to get users by app metadata dorbel user id.
+    })
+    .then(function (user) {
+      logger.debug(user);
```

Comment:
1. Please add a message when logging.
2. Maybe all these internals should not be debugged from a utility ? or at least be ``trace`` instead of ``debug``